# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/js/builtins/weakrefs/finalizationregistry-cleanupCallback-gets-a-microtask.optional.any.js.ini
# Commit: 85883a720dab
# Full Hash: 85883a720dab61e82bcdb7391236dc06427466ee
# Author: moz-wptsync-bot <wptsync@mozilla.com>
# Date: 2021-04-07 15:47:02
# Regressor Bug: 1702830
# File Overlap Count: 1
# Description:
#   Bug 1702830 - [wpt-sync] Update web-platform-tests to 336ec39f06d15863a4af6b2aa6592530b37a5aee, a=testonly
#   
#   MANUAL PUSH: wpt sync bot
#   
#   wpt-head: 336ec39f06d15863a4af6b2aa6592530b37a5aee
# ==============================================================================

diff -r acef12d74007 -r 85883a720dab testing/web-platform/meta/js/builtins/weakrefs/finalizationregistry-cleanupCallback-gets-a-microtask.optional.any.js.ini
--- a/testing/web-platform/meta/js/builtins/weakrefs/finalizationregistry-cleanupCallback-gets-a-microtask.optional.any.js.ini	Fri Apr 02 22:03:46 2021 +0000
+++ b/testing/web-platform/meta/js/builtins/weakrefs/finalizationregistry-cleanupCallback-gets-a-microtask.optional.any.js.ini	Fri Apr 02 22:06:52 2021 +0000
@@ -7,11 +7,11 @@
   [HostCleanupFinalizationRegistry is an implementation-defined abstract operation that is expected to call CleanupFinalizationRegistry(finalizationRegistry) at some point in the future, if possible.]
     expected:
       if (os == "win") and ccov: [FAIL, PASS]
-      if (os == "win") and debug and webrender and not swgl: ["FAIL", "PASS"]
-      if (os == "linux") and not debug and not webrender and (processor == "x86"): ["FAIL", "PASS"]
-      if (os == "linux") and not debug and webrender: ["FAIL", "PASS"]
+      if (os == "linux") and not webrender and (processor == "x86_64") and not debug: ["FAIL", "PASS"]
+      if (os == "linux") and not webrender and (processor == "x86_64") and debug: ["FAIL", "PASS"]
+      if (os == "linux") and webrender and not debug: ["FAIL", "PASS"]
+      if (os == "linux") and not webrender and (processor == "x86"): ["FAIL", "PASS"]
       if (os == "win") and not debug: ["FAIL", "PASS"]
-      if (os == "android") and debug: ["FAIL", "PASS"]
       if (os == "mac") and debug: PASS
       if (os == "mac") and not debug: FAIL
       [PASS, FAIL]